ios/brave-ios/Sources/Brave/Frontend/Sync/SyncAddDevice/SyncAddDeviceDetailView.swift

```diff
@@ -216,7 +216,7 @@ class SyncAddDeviceActionView: UIStackView {
     $0.setTitle(Strings.done, for: .normal)
     $0.titleLabel?.font = .preferredFont(for: .subheadline, weight: .semibold)
     $0.addTarget(self, action: #selector(dismiss), for: .touchUpInside)
-    $0.setTitleColor(.white, for: .normal)
+    $0.setTitleColor(UIColor(braveSystemName: .schemesOnPrimary), for: .normal)
```

Comment:
If you are using `buttonBackground` as the background, you should use `schemesOnPrimary` and not white even if Figma says otherwise. If Figma explicitly uses a different button background color then you can use white assuming the dark mode variant still works